aDataGenerator/amenitiesGenerator.js

```diff
@@ -0,0 +1,116 @@
+const pg = require('pg');
+const key = require('../config.js');
+const fs = require('fs');
+const path = require('path');
+const faker = require('faker');
```

Comment:
You haven't used faker in any of your code, perhaps just get rid of this line.
However, if you decide to use faker it will significantly shorten the data generation script.